triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -63,7 +63,7 @@ object TriggerServiceFixture {
         ledgerPort,
         TimeProviderType.Static,
         Duration.ofSeconds(30))
-      service <- ServiceMain.startServer("localhost", 0, ledgerConfig, dar)
+      service <- ServiceMain.startServer("localhost", 0, ledgerConfig, 4194304, dar)
```

Comment:
Please use the default you specified above instead of duplicating it.

Reply:
✔️